README.md

```diff
@@ -37,14 +37,62 @@ To turn your web pages into Frames, you need to add basic metadata to your page.
 | Property | Description |
```

Comment:
we need a general strategy about backwards compatible (for clients) versioning of changes like this